sass/typography/presets/_bootstrap.scss

```
@@ -0,0 +1,118 @@
+$_scale: (
```

```suggestion
$scale: (
```
This should definitely be public.